x-pack/plugins/lens/public/datasources/form_based/operations/definitions/formula/editor/formula_editor.tsx

```diff
@@ -779,7 +779,7 @@ export function FormulaEditor({
             </div>
 
             <div className="lnsFormula__editorContent">
-              <CodeEditor
+              <CodeEditor // TODO theme provided in context by x-pack/plugins/lens/public/app_plugin/mounter.tsx
```

Comment:
Lens formula is broken , I see this error in the console

<img width="837" alt="image" src="https://github.com/elastic/kibana/assets/17003240/16d30983-e488-48d0-88bc-60653db9499b">

Reply:
Ah thank you! Fixed in  https://github.com/elastic/kibana/pull/159638/commits/0ff4ad74db0e8711d4640244f453a9b9f9c4b347